packages/template-retail-react-app/app/components/display-price/index.jsx

```diff
@@ -0,0 +1,67 @@
+/*
+ * Copyright (c) 2023, Salesforce, Inc.
+ * All rights reserved.
+ * SPDX-License-Identifier: BSD-3-Clause
+ * For full license text, see the LICENSE file in the repo root or https://opensource.org/licenses/BSD-3-Clause
+ */
+
+import React from 'react'
+import PropTypes from 'prop-types'
+import {Skeleton, Text} from '@salesforce/retail-react-app/app/components/shared/ui'
+import {useIntl} from 'react-intl'
+import {useCurrency} from '@salesforce/retail-react-app/app/hooks'
+
+const DisplayPrice = ({
+    basePrice,
+    discountPrice,
+    isProductASet = false,
+    currency,
+    discountPriceProps,
+    basePriceProps,
+    skeletonProps
+}) => {
+    const intl = useIntl()
+    const {currency: activeCurrency} = useCurrency()
+    return (
+        <Skeleton isLoaded={basePrice} display={'flex'} {...skeletonProps}>
+            <Text fontWeight="bold" fontSize="md" aria-label="price" mr={1}>
+                {isProductASet &&
+                    `${intl.formatMessage({
+                        id: 'product_view.label.starting_at_price',
+                        defaultMessage: 'Starting at'
+                    })} `}
+            </Text>
+            {discountPrice > 0 && (
+                <Text as="b" {...discountPriceProps}>
+                    {intl.formatNumber(discountPrice, {
+                        style: 'currency',
+                        currency: currency || activeCurrency
+                    })}
+                </Text>
+            )}
+            <Text
+                as={discountPrice > 0 ? 's' : 'b'}
+                ml={discountPrice > 0 ? 2 : 0}
+                fontWeight={discountPrice ? 500 : 'bold'}
```

Comment:
nit: mixing number values and keywords feels icky

Reply:
I changed to use normal vs bold